mkdocs.yml

```
@@ -217,9 +247,15 @@ markdown_extensions:
   - pymdownx.mark
   - pymdownx.tilde
   - attr_list
+  - md_in_html
   - tables
   # - mdx_truly_sane_lists
 extra:
+  version:
+    provider: mike
+  # - make edgexversion the latest version for repository tree references
+  edgexversion: 'v3.0.0'
```

Must be the vanity minor only tag
```suggestion
  edgexversion: 'v3.0'
```

---

fixed